extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -38,7 +42,8 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		this.initialize(workspaceFolders).catch(e => console.error(e));
 		this.viewId = view;
 		this.prompter = new CodeAdapter();
-
+		this._bookTrustManager = new BookTrustManager(this.books);
+		this._apiWrapper = new ApiWrapper();
```

Comment:
Don't create a new ApiWrapper - use the one in the AppContext created in extension.ts